src/chart/line/lineAnimationDiff.js

```diff
@@ -166,6 +166,7 @@ export default function (
 
     // Diff result may be crossed if all items are changed
     // Sort by data index
+    rawIndices =  Array.from(new Set(rawIndices));
```

Comment:
Please don't use `Array.from` and `Set` because it's not supported in IE. See https://www.echartsjs.com/zh/coding-standard.html

Reply:
OK，I will fix it